src/main/java/edu/kit/ipd/crowdcontrol/objectservice/mail/MailHandler.java

```diff
@@ -39,22 +40,39 @@ public MailHandler(Properties props, Authenticator auth) throws MessagingExcepti
         connectToStore();
```

Comment:
then you have to close it manually! even if there are now exceptions, don't forget the finally block.